[github-actions] v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py fails intermittently

[logs_72353.zip](https://github.com/openthread/openthread/files/6389190/logs_72353.zip)
[packet-verification-low-power-pcaps.zip](https://github.com/openthread/openthread/files/6389191/packet-verification-low-power-pcaps.zip)

```
======================================================================
ERROR: test (__main__.LowPower_7_1_01)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/thread_cert.py", line 275, in tearDown
    self._verify_packets(test_info_path)
  File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/thread_cert.py", line 303, in _verify_packets
    self.verify(pv)
  File "./tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py", line 187, in verify
    pkts.filter_wpan_src64(LEADER) \
  File "/home/runner/work/openthread/openthread/tests/scripts/thread-cert/pktverify/packet_filter.py", line 233, in must_next
    raise errors.PacketNotFound(self.index, self._stop_index)
pktverify.errors.PacketNotFound: ((48, 48), (52, 52))

======================================================================
FAIL: test (__main__.LowPower_7_1_01)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py", line 88, in test
    self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD / 1000))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 1 test in 1.096s

FAILED (failures=1, errors=1)
3: exit
loaded 52 packets from /home/runner/work/openthread/openthread/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks_2.pcap
File "tests/scripts/thread-cert/run_cert_suite.py", line 76, in run_cert
2021-04-28 04:52:38,063 - ERROR - Run test v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks_2 failed, please check the log file: v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks_2.log
Traceback (most recent call last):
  File "tests/scripts/thread-cert/run_cert_suite.py", line 69, in run_cert
    subprocess.check_call(["python3", script],
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['python3', './tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py']' returned non-zero exit status 1.
PASS 1 FAIL 0 ./tests/scripts/thread-cert/v1_2_LowPower_5_3_01_SSEDAttachment.py
PASS 0 FAIL 1 ./tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py
PASS 1 FAIL 0 ./tests/scripts/thread-cert/v1_2_LowPower_6_1_07_PreferringARouterOverAReed.py
PASS 1 FAIL 0 ./tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py
PASS 1 FAIL 0 ./tests/scripts/thread-cert/v1_2_LowPower_test_forward_tracking_series.py
Error: Process completed with exit code 1.
```

--- a/tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py
+++ b/tests/scripts/thread-cert/v1_2_LowPower_7_1_01_SingleProbeLinkMetricsWithEnhancedAcks.py
@@ -84,8 +84,8 @@ class LowPower_7_1_01(thread_cert.TestCase):
 
         leader_addr = self.nodes[LEADER].get_ip6_address(ADDRESS_TYPE.LINK_LOCAL)
 
-        # Step 3 - Verify connectivity by instructing each device to sending an ICMPv6 Echo Request to the DUT
-        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD / 1000))
+        # Step 3 - Verify connectivity by instructing each device to send an ICMPv6 Echo Request to the DUT
+        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD * 2 / 1000))
         self.assertTrue(self.nodes[SSED_1].ping(leader_addr))
         self.simulator.go(5)
 

